core/Makefile

```diff
@@ -151,6 +151,16 @@ protobuf: $(PROTO_SRCS)
 
 modules: protobuf $(MODULE_OBJS) $(MODULE_LIBS)
 
+# Generate version string from the current status of the git working copy
+VERSION := $(shell git describe --abbrev=4 --dirty --always --tags 2>/dev/null)
+
+# Default if previous command fails (e.g., when building from a tarball)
+DEFAULT_VERSION := v0.2.1
```

Comment:
Currently, the only tag in the git repository is v0.2.0.  This is larger than that.  I imagined maintainers add tags to some versions from time to time.  For example, if the next tag is v0.3.0, then the Makefile will have to be modified to have a default version number larger than v0.3.0 like v0.3.1.  So v0.2.1 would mean bessd is not built from git, but the source is more fresh than v0.2.0, and older than the hypothetical v0.3.0.  

However, I understand that updating the Makefile after every "release" is an extra burden.  So I'm OK with setting DEFAULT_VERSION to unknown if you accept that.